RevenueCatUI/CustomerCenter/Extensions/CustomerInfo+ActiveTransaction.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfo+ActiveTransaction.swift
+//
+//  Created by Facundo Menzella on 14/4/25.
+
+import RevenueCat
+
+extension CustomerInfo {
+
+    func findActiveTransaction() -> Transaction? {
```

Comment:
I wonder if we can find a better name, since this is a very particular implementation for the customer center